modules/file-rotation/file-rotation.c

```diff
@@ -22,29 +22,46 @@
  */
 
 #include "file-rotation.h"
+#include "modules/http/http-signals.h"
+#include "modules/affile/file-signals.h"
 #include "driver.h"
 
-struct _FileRotationPlugin
+static void
+_slot_append_file_rotation_request(FileRotationPlugin *self, FileFlushSignalData *data)
 {
-  LogDriverPlugin super;
-  gsize size;
-};
+
+  g_string_append_printf(self->filename, "%s", data->filename->str);
```

Comment:
I'd just add a `msg_trace()` here so you can test the slot by starting syslog-ng with `-Fdevt`.
There is no need to append the filename, we can just go with the msg_trace and nothing more for this PR.

Reply:
ok sure